editor/editor_node.cpp

```diff
@@ -3858,18 +3858,18 @@ void EditorNode::register_editor_types() {
 	GDREGISTER_CLASS(EditorScript);
 	GDREGISTER_CLASS(EditorSelection);
 	GDREGISTER_CLASS(EditorFileDialog);
-	GDREGISTER_VIRTUAL_CLASS(EditorSettings);
+	GDREGISTER_ABSTRACT_CLASS(EditorSettings);
```

Comment:
I'm not sure I understand why these editor types are defined as abstract (and were defined as virtual before). What's abstract in `EditorSettings` or `EditorResourcePreview`? Is it just to prevent them from being instantiated?